app/views/player.py

```diff
@@ -9,13 +9,21 @@
 from rest_framework.utils import json
 
 from app.mail import send_tag_email, send_stun_email
-from app.models import Player, PlayerRole, Tag, SupplyCode, Modifier, ModifierType, Spectator, Moderator
+from app.models import Player, PlayerRole, Tag, TagType, SupplyCode, Modifier, ModifierType, Spectator, Moderator, Email, EmailRule
 from app.util import most_recent_game, running_game_required, player_required, get_game_participants, game_required, \
     participant_required
 from app.views.forms import ReportTagForm, ClaimSupplyCodeForm, MessagePlayersForm
 
 from pytz import timezone
 
+import io
+from django.http import FileResponse
+from reportlab.platypus import Table, TableStyle, SimpleDocTemplate
```

Comment:
This introduces dependencies. I don't know exactly where we put our mandatory dependencies, but there is definitely a place for it. Perhaps @pkrshnan knows more about that.